tools/fw-logger/fw-logs-formating-options.h

```diff
@@ -33,15 +33,15 @@ namespace fw_logger
         bool get_event_data(int id, fw_log_event* log_event_data) const;
         bool get_file_name(int id, std::string* file_name) const;
         bool get_thread_name(uint32_t thread_id, std::string* thread_name) const;
-        std::unordered_map<std::string, std::vector<std::string>> get_enums() const;
+        std::unordered_map<std::string, std::vector<std::pair<int, std::string>>> get_enums() const;
```

Comment:
Not clear at all what this map maps.
Would be nice to have typedefs here:

```
typedef std::string enum_name_t;
typedef std::string enum_value_name_t;
typedef int enum_value_id_t;
typedef std::pair< enum_value_id_t, enum_value_name_t > enum_value_id_name_t;
typedef std::vector< enum_value_id_name_t > enum_value_id_name_arr_t;
typedef std::unordered_map< enum_name_t, enum_value_id_name_arr_t > enum_name_to_values_t;
```

Reply:
Also, might want to return a 'const &'

Reply:
I made minor changes, mostly to the refactored segment.